### ISSUE TITLE
Implement the ProjectSectionView

### ISSUE THREAD

**Issue by csjx** [NCEAS/metacatui#815]
A Project Section is a generic section view with a `TOCView` and a `MarkdownView` as subviews. Create this view with its subviews.

This is a subtask of https://github.com/NCEAS/project-papers/issues/35

**Comment by laurenwalker** [comment]
I believe this is done now.

I reworked how the tabs are done in the views in the latest commit - 49eedc3e1b0fe7541ab22774452e75a8566a49c1

The tabs were originally being rendered by each ProjectSectionView. But the tab elements will not be contained within each section view. They will be in the navigation bar outside of each section, just below the header. Each view should only render and manipulate elements that are inside the view `el`. 

So I moved the tabs to the `project.html` template. Each section view is agnostic to the navigation outside of it (i.e. the section views don't know that tabs or other links to it exist).
